web-ui/view/timestamps_durations.ml

```diff
@@ -6,6 +6,14 @@ let to_iso8601 (tt : float) =
   let ts = Timedesc.of_timestamp_float_s tt in
   Timedesc.to_iso8601 @@ Option.get ts
 
+let to_ts_string (tt : float) =
+  let ts = Timedesc.of_timestamp_float_s tt in
+  Timedesc.to_string
+    ~format:
+      "{mon:Xxx} {day:0X} {hour:0X}:{min:0X} \
+       {tzoff-sign}{tzoff-hour:0X}:{tzoff-min:0X}"
+  @@ Option.get ts
```

Comment:
Are we sure to have an `optionè here by construction? (Prefer to ask for nothing than having errors at runtime)